src/utils/resourceRBAC.tsx

```
@@ -24,22 +26,28 @@ function useAccessReviews(resource: Resource[]) {
       setLoading(false);
     };
 
     checkRBAC();
-  }, []);
+  }, [resource]);
 
   return { userRBAC, loading };
 }
 
-export default useAccessReviews;
+export default useAccessReviews;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)